articles/communication-services/quickstarts/email/includes/send-email-net.md

```diff
@@ -413,4 +413,36 @@ emailMessage.Attachments.Add(emailAttachment);
 
 You can download the sample app demonstrating this action from [GitHub](https://github.com/Azure-Samples/communication-services-dotnet-quickstarts/tree/main/SendEmailAdvanced/SendEmailWithAttachments)
 
+### Throw an exception when email sending tier limit is reached
 
+There are per minute and per hour limits to the amount of emails you can send using the Azure Communication Email Service. When you have reached these limits, any further `SendAsync` calls will recieve a `429: Too Many Requests` response. By default, the SDK is configured to retry these requests after waiting a certain period of time. We recommend you [set up logging with the Azure SDK](https://learn.microsoft.com/en-us/azure/developer/java/sdk/logging-overview) to capture these response codes.
+
+If setting up logging is not an option, you can manually define a custom policy as shown below. 
+
+```csharp
+using Azure.Core.Pipeline;
+
+public class Catch429Policy : HttpPipelineSynchronousPolicy
+{
+    public override void OnReceivedResponse(HttpMessage message)
+    {
+        if (message.Response.Status == 429)
```

Comment:
Perhaps we should also surface the error code/message here - that'll give an indication of whether it was throttled due to reaching perminute or perhour limit.